### DISCUSSION TITLE
How to boot over PCIe USB3 on an old BIOS USB2 motherboard like my GA-E7AUM-DS2H

### DISCUSSION THREAD

**Discussion by vinceFR7528** [CloverHackyColor/CloverBootloader#535] [General]
Hi,

My old GA-E7AUM-DS2H BIOS mobo has only USB2 and when it boots on a USB live disk it uses the fullspeed 12Mb (not Hi-Speed, strange?) so booting a linux ISO takes a long long time. I added a PCIe USB3 extension card, this card is seen by the bios but not used so the disk connected to it is not seen from the boot loader. I installed CLOVER 5148 and tried to load the XhciDxfe.efi driver, but it doesn't work.

Is there a workaround to let CLOVER boot a live OS on the USB3 extension card?

**Comment by Morbius01** [comment]
Interestingly, I have an old Gigabyte MB of about the same age (year 2008 for mine).
Just like you, I have installed a PCIe USB3 extension card to get higher speeds when using a USB drive. The difference with you is that my BIOS cannot see the PCIe card.
I have also installed an SSD through a NVMe to PCIe adaptor. And I have installed several OSs on this SSD on which I boot through Clover.

Now, back to your question.

1. If your current OS can see the external disk connected to one of the USB3 ports, I don’t see why you would not be able to install an OS on this external disk and boot on it through Clover. You just need to try...

2. If your concern is to boot with faster speeds, then you may like to consider to do what I did, that is, acquire an NVMe SSD and a NVMe to PCIe adaptor, install it on your MB, and then install your OS(s) through Clover. You will then get Warp speeds ! (on my system, Windows 11 boots in 13 s [thirteen seconds] and Kali Linux in 15 s from the Clover menu !) In addition, you will get better performances if your system disk is on a NVMe SSD.

See my [post](https://github.com/CloverHackyColor/CloverBootloader/discussions/491).

Whichever solution you opt for, keep in mind that you need to create a Clover key first, and then install your OS from this Clover key (and not the other way round).

**Comment by vinceFR7528** [comment]
So, it should be possible but the XhciDxfe.efi driver is not working.

**Comment by Morbius01** [comment]
Frankly, who cares about this 'XhciDxfe.efi driver (I don't even have it in my Clover key !) ?

Clover works, and it works well !
If you want to make Clover work, just follow the instructions (I can help with that if you request for it).
